[2023-03-16T04:30:49.234+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Spotify_Data_ETL.ValidateAndLoadArtistFact.ValidateArtistFact manual__2023-03-16T04:25:58.412323+00:00 [queued]>
[2023-03-16T04:30:49.249+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Spotify_Data_ETL.ValidateAndLoadArtistFact.ValidateArtistFact manual__2023-03-16T04:25:58.412323+00:00 [queued]>
[2023-03-16T04:30:49.250+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-16T04:30:49.252+0000] {taskinstance.py:1280} INFO - Starting attempt 5 of 5
[2023-03-16T04:30:49.252+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-16T04:30:49.269+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): ValidateAndLoadArtistFact.ValidateArtistFact> on 2023-03-16 04:25:58.412323+00:00
[2023-03-16T04:30:49.277+0000] {standard_task_runner.py:55} INFO - Started process 5708 to run task
[2023-03-16T04:30:49.283+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Spotify_Data_ETL', 'ValidateAndLoadArtistFact.ValidateArtistFact', 'manual__2023-03-16T04:25:58.412323+00:00', '--job-id', '997', '--raw', '--subdir', 'DAGS_FOLDER/spotifydag.py', '--cfg-path', '/tmp/tmpm7mtsy8t']
[2023-03-16T04:30:49.285+0000] {standard_task_runner.py:83} INFO - Job 997: Subtask ValidateAndLoadArtistFact.ValidateArtistFact
[2023-03-16T04:30:49.384+0000] {task_command.py:388} INFO - Running <TaskInstance: Spotify_Data_ETL.ValidateAndLoadArtistFact.ValidateArtistFact manual__2023-03-16T04:25:58.412323+00:00 [running]> on host 18c42e4ef76b
[2023-03-16T04:30:49.697+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ananta Moharana
AIRFLOW_CTX_DAG_ID=Spotify_Data_ETL
AIRFLOW_CTX_TASK_ID=ValidateAndLoadArtistFact.ValidateArtistFact
AIRFLOW_CTX_EXECUTION_DATE=2023-03-16T04:25:58.412323+00:00
AIRFLOW_CTX_TRY_NUMBER=5
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-16T04:25:58.412323+00:00
[2023-03-16T04:30:49.714+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/transform.py", line 69, in validate_data
    if data_table.duplicated.any():
AttributeError: 'function' object has no attribute 'any'
[2023-03-16T04:30:49.731+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=Spotify_Data_ETL, task_id=ValidateAndLoadArtistFact.ValidateArtistFact, execution_date=20230316T042558, start_date=20230316T043049, end_date=20230316T043049
[2023-03-16T04:30:49.741+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 997 for task ValidateAndLoadArtistFact.ValidateArtistFact ('function' object has no attribute 'any'; 5708)
[2023-03-16T04:30:49.778+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-16T04:30:49.818+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
